### ISSUE TITLE
Average time per year stat

### ISSUE THREAD

**Issue by lauracowen** [fraz3alpha/running-challenges#192]
User emailed to ask if we could add average time per year.

parkrun's default stats show average (mean) time overall and best time per year but not average time per year.

**Comment by fraz3alpha** [comment]
For reference, The standard stats provided in the main box at the top look like this (these are my stats):

<img width="429" alt="Screenshot 2019-11-17 at 09 32 08" src="https://user-images.githubusercontent.com/2786011/69005654-2e143800-091d-11ea-858c-3b7c782dca70.png">

HQ have been tolerant/fine with us adding extra stuff to the page via the extension, but from a few comments i've heard they've thought hard about the stats they have provided, and have reasons for displaying - or not displaying, certain ones. This, as well as keeping our additions separate, I'd rather not go adding to their tables and cloud what we add / vs what is there already.

In summary, I don't think we should add this.
